docs/sources/developers/plugins/publish-a-plugin/_index.md

```diff
@@ -1,11 +1,19 @@
 ---
-description: Topics related to plugin publishing
 title: Publish a plugin
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - publish
+  - publishing
+description: An index to documentation related to plugin publishing.
+weight: 400
 ---
 
 # Publish a plugin
 
 This section contains topics related to publishing Grafana plugins.
 
-- [Publishing and signing criteria]({{< relref "../publishing-and-signing-criteria.md">}})
-- [Publish or update a plugin]({{< relref "../publish-or-update-a-plugin.md">}})
+- [Publish or update a plugin]({{< relref "./publish-or-update-a-plugin.md" >}})
```

Comment:
Both in the ToC and here I would list in the following order:
- Package a plugin
- Publishing and signing criteria
- Publish or update a plugin

Reply:
Done.